spec/features/visitors/password_recovery_spec.rb

```diff
@@ -2,9 +2,11 @@
 
 feature 'Password Recovery' do
   def reset_password_and_sign_back_in(user, password = 'a really long password')
-    fill_in 'New password', with: password
+    fill_in t('forms.passwords.edit.labels.password'), with: password
     click_button t('forms.passwords.edit.buttons.submit')
-    fill_in_credentials_and_submit(user.email, password)
+    fill_in 'Email', with: user.email
+    fill_in 'user_password', with: password
+    click_button t('links.next')
```

Comment:
What was the reason for replacing the `fill_in_credentials_and_submit` helper with its contents?